hw/ip/otbn/util/shared/toolchain.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python3
```

Comment:
No need for that, it's not a stand-alone script.

Reply:
Good point - I think I copied the header from the wrong file. Thanks!